xmind/tests/test_workbook_mixin_element.py

```diff
@@ -0,0 +1,135 @@
+import unittest
+from unittest.mock import patch, Mock, MagicMock, PropertyMock
+from xmind.core.mixin import WorkbookMixinElement
+import logging
+from . import base
+
+
+class WorkbookMixinElementTest(base.Base):
+    """WorkbookMixinElement test"""
+    LOGGER = logging.getLogger('WorkbookMixinElementTest')
+
+    def setUp(self):
+        super(WorkbookMixinElementTest, self).setUp()
+
+        self._init_method = self._init_patch_with_name('_init', 'xmind.core.Element.__init__')
+
+        self._ownerWorkbook = MagicMock()
+
+        if self._testMethodName != "test_regist_owner_workbook" and self._testMethodName != "test_regist_none_owner_workbook":
```

Comment:
You shouldn't do some specific steps needed only for specific test in setUp method.

SetUp has to do general stuff. Whatever is specific for your test has to be done in tests itself,